_includes/accordion.html

```diff
@@ -12,6 +12,7 @@
       aria-controls="{{ include.accordion_id }}"
     >
       {{ include.title }}
+      <a href="#{{ include.accordion_id }}" class="accordion-link">#</a>
```

Comment:
What the ICT Baseline page does is to include `aria-label="copy"` and `aria-describedby={usa-accordion__button.id}`
This way it shows as "copy {FAQ title}" in the Link rotor, and reads as "copy" (without duplicating the FAQ title) in the flow of the page.

Reply:
I think you could go back to `anchor`, as you have the added context where needed.